mesa/space.py

```diff
@@ -4,9 +4,13 @@
 
 Objects used to add a spatial component to a model.
 
-Grid: base grid, a simple list-of-lists.
-SingleGrid: grid which strictly enforces one object per cell.
-MultiGrid: extension to Grid where each cell is a set of objects.
+Grid: base grid, it creates a rectangular grid.
```

Comment:
Should be `base grid, which creates a rectangular grid`.

Reply:
👍